ts/screens/onboarding/OnboardingServicesPreferenceScreen.tsx

```diff
@@ -103,6 +102,7 @@ const OnboardingServicesPreferenceScreen = (props: Props): ReactElement => {
       store.getState()
     );
     onContinue(isFirstOnboarding);
+    IOToast.hideAll();
```

Comment:
Since we are in a functional component, what do you think about replacing `IOToast` ref with `useIOToast` hook?

Reply:
yes, good suggestion